memstore/memstore.go

```diff
@@ -31,19 +31,27 @@ type TableShardMemoryUsage struct {
 	PrimaryKeyMemory uint                                 `json:"pk"`
 }
 
+type TableSchemaReader interface {
```

Comment:
might want to move this interface to common, so broker will not depend on memstore

Reply:
sg. will move some interface def in metastore to common as well